irods_docker_files/run_tests_in_parallel.py

```diff
@@ -85,7 +85,7 @@ def to_os_name(docker_image_name):
     if 'ubuntu_16' in name: return 'Ubuntu_16'
     if 'ubuntu_18' in name: return 'Ubuntu_18'
     if 'ubuntu_20' in name: return 'Ubuntu_20'
-    if 'centos_7' in name : return 'Centos_7'
+    if 'centos_7' in name : return 'Centos linux_7'
```

Comment:
hmmm, can we use clean consistent naming in our own stuff - and only deal with the silly spaces that are produced by the commands from the OS itself when we have to?

Reply:
We can, but that would require changing more things. This particular change just makes sure the log is placed with the other output files. If I left this as Centos_7, two directories would appear and confuse people.

We can handle the naming in a later PR if that is ok.

Reply:
sure thing - just want to keep our own stuff predictable/sane.